File: tests/src/main/java/com/github/klee0kai/test/mowgli/body/Blood.java
package com.github.klee0kai.test.mowgli.body;

import java.awt.*;
import java.util.UUID;

public class Blood {

    public UUID uuid = UUID.randomUUID();

    public Color color = null;

    public Blood(Color color) {
        this.color = color;
    }
}

File: tests/src/commonMain/kotlin/com/github/klee0kai/test/mowgli/body/Blood.kt
@file:OptIn(ExperimentalUuidApi::class)

package com.github.klee0kai.test.mowgli.body

import kotlin.uuid.ExperimentalUuidApi
import kotlin.uuid.Uuid

class Blood(color: Int?) {

    var uuid: String = Uuid.random().toString()

    var color: Int? = null

    init {
        this.color = color
    }
}
